odoo/service/model.py

```diff
@@ -159,5 +159,6 @@ def execute(db, uid, obj, method, *args, **kw):
         check_method_name(method)
         res = execute_cr(cr, uid, obj, method, *args, **kw)
         if res is None:
-            _logger.info('The method %s of the object %s can not return `None` !', method, obj)
+            _logger.info('The method %s of the object %s should not return `None` !', method, obj)
+            res = True
```

Comment:
Maybe replace None by a string could be better. Use `True` can be confusing.

`res = 'None'`